test-graalvm/common/build.gradle

```diff
@@ -11,3 +11,9 @@ dependencies {
     }
     implementation(mnSerde.micronaut.serde.jackson)
 }
+
+graalvmNative {
+    metadataRepository {
+        uri("https://github.com/oracle/graalvm-reachability-metadata/releases/download/0.2.7/graalvm-reachability-metadata-0.2.7.zip")
```

Comment:
Should create an issue to remind us to remove this line when a new Gradle native image plugin is out